packages/frameworks/react/src/date-picker/use-date-picker.ts

```diff
@@ -19,6 +22,15 @@ export interface UseDatePickerProps
    * The value of the date picker
    */
   value?: string[]
+  /**
+   * The minimum date for the date picker in the format yyyy-mm-dd
+   */
+  min?: string
+
+  /**
+   * The maximum date for the date picker in the format yyyy-mm-dd
+   */
+  max?: string
```

Comment:
@cschroeter @segunadebayo 
you may want to reference this change as breaking instead of fix as old calendarDate do not work anymore

----
btw that's too bad it was working quite well and zag is using calendar date
maybe we could use an union here ?

Reply:
+1